### ISSUE TITLE
Particularly bad queries

### ISSUE THREAD

**Issue by sandrine-muller-research** [NCATSTranslator/Feedback#644]
These are queries that currently yield to a lot of overly generic and promicuous compound (or sporadically other types of never show compounds) at a score >4:

1. What drugs may treat:[Peptic Ulcer Perforation](http://purl.obolibrary.org/obo/MONDO_0004260):
https://ui.test.transltr.io/main/results?l=Peptic%20Ulcer%20Perforation&i=MONDO:0004260&t=0&q=6999a612-9767-4a47-802d-172c40e8ac74

**Comment by gglusman** [comment]
Scanning through the list, I see two main types of patterns leading to bad answers:
1) The "part of / has part" weak reasoning we captured with the #transitive label. (in this case, frequently via Cimetidine)
2) Paths via the very promiscuous ALB (albumin) gene. (in this case, also via the GAST gene)

**Comment by sandrine-muller-research** [comment]
yes, that is likely what is happening here. my idea through this ticket would be to identify a bunch of bad queries as the "worse" we could do to try understanding why and also get performance metrics for worse case scenarii.

**Comment by sierra-moxon** [comment]
from TAQA:
we note that Cimetidine shows up as a middle node in an inference path frequently, that this could be collapsed potentially into one result vs. many.  we also note that "cation" is one level of too generic that we see in these results.

in the UI group, they have seen exact same nodes but different predicates, but not collapse on "same central node" kinds of results like the Cimetidine example.

action:
"cation"/"anion" - put that on a blocklist

**Comment by sstemann** [comment]
related to #591 and #617 maybe we can merge the tickets and assign to the appropriate development team
